config/example/ir.env

```diff
@@ -0,0 +1,165 @@
+# Logger level: one of "debug", "info" (default), "warn", "error", "dpanic", "panic", "fatal"
```

Comment:
For storage node, we've decided to provide docs in single place, e.g. yaml. I suggest to do it for IR as well.

Reply:
Fixed https://github.com/nspcc-dev/neofs-node/pull/1364/commits/4ac0dccf081df1c25885e5c81c7419ed18a54984